index.html

```diff
@@ -83,9 +83,9 @@ <h2>Multi-Post Stories</h2>
                             </a>
                         </li>
                     </ul>
-                    <a href="#">
-                        <img src="images/Enable.png" alt="seeProjectButton">
-                    </a>
+                    <div class="alingButton">
+                        <input class="btnPrimary" type="button" value="See Project">
```

Comment:
I think would be better if you use `<Button>` instead of  `<input>` since it's clear and more specific, Don't you think?

Reply:
Completely agree with you! 🤓 

Thanks for your time, reviewing my code! I will apply your suggestion! 🤝